ios/RNCWebView.m

```diff
@@ -162,51 +169,8 @@ - (WKWebViewConfiguration *)setUpWkWebViewConfig
 
   // Shim the HTML5 history API:
   [wkWebViewConfig.userContentController addScriptMessageHandler:[[RNCWeakScriptMessageDelegate alloc] initWithDelegate:self]
-                                                            name:HistoryShimName];
```

Comment:
This cut off a piece of the function, could you restore this?

Reply:
Now resolved – sorry for the trouble!